File: handlers/topics.py
from aiogram import Bot
from aiogram.types import InlineKeyboardMarkup, InlineKeyboardButton

def get_topic_setup_keyboard(auto_pr: bool = False):
    btn_auto_pr = InlineKeyboardButton(text=f"Auto PR: {'ON' if auto_pr else 'OFF'}", callback_data=f"toggle_pr_{int(auto_pr)}")
    btn_proceed = InlineKeyboardButton(text="Proceed ➔", callback_data="proceed_setup_new")
    btn_cancel = InlineKeyboardButton(text="Cancel / Delete Topic", callback_data="cancel_setup")

    return InlineKeyboardMarkup(inline_keyboard=[
        [btn_auto_pr],
        [btn_proceed],
        [btn_cancel]
    ])

async def send_topic_setup_menu(bot: Bot, chat_id: int, message_thread_id: int, auto_pr: bool = False):
    await bot.send_message(
        chat_id=chat_id,
        message_thread_id=message_thread_id,
        text="Please configure this topic:",
        reply_markup=get_topic_setup_keyboard(auto_pr)
    )

File: src/handlers/topics.ts
import { InlineKeyboardMarkup } from '@grammyjs/types';
import { callApi } from '../api';

export function getTopicSetupKeyboard(autoPr: boolean = false): InlineKeyboardMarkup {
  const btnAutoPr = { text: `Auto PR: ${autoPr ? 'ON' : 'OFF'}`, callback_data: `toggle_pr_${autoPr ? '1' : '0'}` };
  const btnProceed = { text: "Proceed ➔", callback_data: "proceed_setup_new" };
  const btnCancel = { text: "Cancel / Delete Topic", callback_data: "cancel_setup" };

  return {
    inline_keyboard: [
      [btnAutoPr],
      [btnProceed],
      [btnCancel]
    ]
  };
}

export async function sendTopicSetupMenu(chatId: number, messageThreadId: number, autoPr: boolean = false) {
  await callApi('sendMessage', {
    chat_id: chatId,
    message_thread_id: messageThreadId,
    text: "Please configure this topic:",
    reply_markup: getTopicSetupKeyboard(autoPr)
  });
}
